### ISSUE TITLE
When creating image using CLI, ide.disk is selected even though headnode's HD is sd

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#314]
**Reported by bli on 14 Jun 2007 07:07 UTC**
`build_oscar_image_cli` has the following code snippet:

```
my @df_lines = `df /`;
my $disk_type = "ide";
$disk_type = "scsi" if (grep(/\/dev\/sd/,(@df_lines)));
```

`df /` returns:

```
Filesystem           1K-blocks      Used Available Use% Mounted on
/dev/mapper/VolGroup00-LogVol00
                       7459216   5756680   1317512  82% /
```

Which would cause `ide.disk` to be chosen as default, which is wrong.  A better check needs to be implemented.

**Comment by dikim33** [comment]
**Comment by valleegr on 26 Aug 2007 03:31 UTC**
Note that this is because LVM is used. There is no code in OSCAR to support LVM (i do not know if systemimager fully supports LVM or not).

**Comment by dikim33** [comment]
**Comment by anandtt on 10 Jun 2009 16:59 UTC**
(In [8557]) This check-in fixes #314 - wrond disk type returned
